sys-auth/libyubikey/libyubikey-1.13.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Low level C SDK for the Yubico YubiKey authentication device"
+DESCRIPTION="This package make up the low-level C software development kit for the Yubico YubiKey authentication device."
+HOMEPAGE="https://developers.yubico.com/yubico-c/"
+SOURCE_URI="https://developers.yubico.com/yubico-c/Releases/libyubikey-1.13.tar.gz"
+CHECKSUM_SHA256="04edd0eb09cb665a05d808c58e1985f25bb7c5254d2849f36a0658ffc51c3401"
+REVISION="1"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="Yubico AB"
+
+ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libyubico = $portVersion
+	cmd:modhex = $portVersion
+	cmd:ykparse = $portVersion
+	cmd:ykgenerate = $portVersion
+	lib:libyubikey
+	"
+
+PROVIDES_devel="
+	libyubikey_devel = $portVersion
+	devel:libyubikey
+	"
+
+REQUIRES="
+    haiku >= $haikuVersion
+    "
+
+BUILD_REQUIRES="
+"
+
+BUILD_PREREQUIRES="
+    haiku_devel >= $haikuVersion
+    cmd:gcc
+    cmd:make
+    cmd:awk
+    "
+
+BUILD()
+{
+    runConfigure ./configure --disable-dependency-tracking
+    make $jobArgs
+}
+
+INSTALL()
+{
+    make install
+
+    prepareInstalledDevelLibs libyubikey
+   	packageEntries devel $developDir
```

Comment:
Could you please change this to:
```
	rm $libDir/libyubikey.la
	prepareInstalledDevelLib libyubikey
	fixPkgconfig
	packageEntries devel \
		$developDir
```
(I'm not sure yet for the **`rm $libDir/libyubikey.la`** because I don't know if that file is being generated.)